rsclone/src/app/components/main-page/theme-switcher/theme-switcher.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, OnInit } from '@angular/core';
+import { ThemeService } from 'src/app/theme/theme.service';
+
+@Component({
+  selector: 'app-theme-switcher',
+  templateUrl: './theme-switcher.component.html',
+  styleUrls: ['./theme-switcher.component.scss']
+})
+export class ThemeSwitcherComponent implements OnInit {
+  lightTheme: boolean = this.themeService.isLightTheme();
+  darkTheme: boolean = this.themeService.isDarkTheme();
+
+  constructor(private themeService: ThemeService) { }
+
+  ngOnInit() {
+    this.setStartTheme();
+  }
+
+  setStartTheme(): void {
+    if (localStorage.getItem('theme') === 'light') {
+      this.setLightTheme();
+    } else if (localStorage.getItem('theme') === 'dark') {
+      this.setDarkTheme();
+    }
```

Comment:
`localStorage.getItem('theme') === 'light' ? this.setLightTheme() : this.setDarkTheme();`

Reply:
fixed